src/components/NationalText/utils.ts

```diff
@@ -0,0 +1,62 @@
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last } from 'lodash';
+import moment from 'moment';
+import { formatPercent, formatDecimal } from 'common/utils';
+
+// Fix all uses of !
+
+const getPercentChange = (series: any[]): number => {
+  const lastTwoWeeks = series.slice(-14);
+  const difference = last(lastTwoWeeks)! - lastTwoWeeks[0]!;
+  const percentChange = difference / lastTwoWeeks[0]!;
+  return percentChange;
+};
+
+const getDescriptiorCopy = (percentChange: number): string => {
+  if (percentChange > 0) {
+    return 'increased ';
+  } else return 'decreased ';
+};
+
+const getTotalCasesCopy = (summedRawCases: number): string => {
+  const divided = summedRawCases / 1000000;
```

Comment:
Consider defining a constant for `1000000` to make this easier to read, or use `1_000_000` (which is another way to write 1000000, I found out about this recently!)

```ts
const ONE_MILLION = 1000000;

// this also works:
const divided = summedRawCases / 1_000_000;
```